utils/vscode/package.json

```diff
@@ -18,7 +18,10 @@
     "languages": [
       {
         "id": "carbon",
-        "aliases": [],
+        "aliases": [
+          "Carbon",
+          "Carbon"
```

Comment:
Do we need this to be listed twice?

Reply:
Even I was wondering about this, because this was autogenerated by the vscode's yo-generator

Reply:
I just tried it listing once, it is also working, but without the list, it is not working, So I am just mentioning it once inside the list in the next update.